lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -64,10 +73,17 @@ class OrderView(APIView):
     """ Retrieve order details. """
 
     authentication_classes = (SessionAuthentication,)
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (IsAuthenticatedOrActivationOverridden,)
```

Comment:
Ah, I guess the question I should have been asking is does the `IsAuthenticated` permission class factor in user activation? Otherwise, why do we need to check for the activation override here?